app/_src/guides/excluding-traffic-from-transparent-proxy-redirection.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Excluding Traffic from Transparent Proxy
+---
+
+{% assign docs = "/docs/" | append: page.version %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign tproxy = site.data.tproxy %}
+
+{% capture path-reference-tproxy-title %}{{ docs }}{{ tproxy.paths.reference.configs.tproxy }}#{{ tproxy.ids.reference.configs.tproxy.title }}{% endcapture %}
+{% capture path-customize-k8s-title %}{{ docs }}{{ tproxy.paths.guides.customize-config.k8s }}#{{ tproxy.ids.guides.customize-config.k8s.title }}{% endcapture %}
+{% capture path-customize-uni-title %}{{ docs }}{{ tproxy.paths.guides.customize-config.uni }}#{{ tproxy.ids.guides.customize-config.uni.title }}{% endcapture %}
+{% capture path-method-k8s-configmap %}{{ docs }}{{ tproxy.paths.guides.customize-config.k8s }}#{{ tproxy.ids.guides.customize-config.k8s.methods.configmap }}{% endcapture %}
+{% capture path-method-k8s-runtime %}{{ docs }}{{ tproxy.paths.guides.customize-config.k8s }}#{{ tproxy.ids.guides.customize-config.k8s.methods.runtime }}{% endcapture %}
+{% capture path-method-k8s-annotations %}{{ docs }}{{ tproxy.paths.guides.customize-config.k8s }}#{{ tproxy.ids.guides.customize-config.k8s.methods.annotations }}{% endcapture %}
+{% capture path-method-uni-env %}{{ docs }}{{ tproxy.paths.guides.customize-config.uni }}#{{ tproxy.ids.guides.customize-config.uni.methods.env }}{% endcapture %}
+{% capture path-method-uni-cli %}{{ docs }}{{ tproxy.paths.guides.customize-config.uni }}#{{ tproxy.ids.guides.customize-config.uni.methods.cli }}{% endcapture %}
+{% capture path-integrating-tproxy-title %}{{ docs }}{{ tproxy.paths.guides.integrate-tproxy }}#{{ tproxy.ids.guides.integrate-tproxy.title }}{% endcapture %}
+{% capture path-step-install-tproxy %}{{ docs }}{{ tproxy.paths.guides.integrate-tproxy }}#{{ tproxy.ids.guides.integrate-tproxy.steps.install-tproxy }}{% endcapture %}
+
+{% assign configuration-in-configmap = "Configuration in ConfigMap <sup>(experimental)</sup>" %}
```

Comment:
Do we need all the options here? Should we just go through the most common path and link somewhere that ConfigMap is also an option?